SRC/NewPortfolio/about.css

```diff
@@ -0,0 +1,122 @@
+body {
+  width: 100%;
+  height: auto;
+  /* border: 1px solid red; */
+  padding: 10%;
```

Comment:
this does something weird. look at this again. i'd remove it. 
also add `margin: 0;` to get rid of the default margin.